### PR TITLE
Caj880 pin heads

### PR DESCRIPTION
**Pull request by camilleion**
Some bugs exist however it is important to show the stakeholder added features

### CODE REVIEW

File: src/components/TimelineComponent.tsx [hunk 1]
```diff
@@ -972,3 +977,10 @@ export default class TimelineComponent
     d3.select(caller).classed('active', false);
   }
 }
+
+/**
+ * an enum to help track which chart is being displayed
+ */
+export enum ViewType {
```
Comment (braunsonm) [inline src/components/TimelineComponent.tsx]: Nice use of an enum

File: src/components/TimelineComponent.tsx [hunk 2]
```diff
@@ -420,9 +415,10 @@ export default class TimelineComponent
         .domain([minDate, maxDate])
         .range([0, 50 * csvData.length]);
 
-    x = d3.scaleLinear()
-        .domain([0, ordinals.length])
-        .rangeRound([0, width]);
+    x = d3.scaleBand()
+        // .padding(1)
```
Comment (braunsonm) [inline src/components/TimelineComponent.tsx]: padding was added as a commented out line?

File: src/components/TimelineComponent.tsx [hunk 3]
```diff
@@ -810,11 +806,12 @@ export default class TimelineComponent
    * @param {any} selection: the selection for the object to draw
    */
   drawEventMagnitude(selection: any): void {
+    const spacing = 48.5; // this needs to change
```
Comment (braunsonm) [inline src/components/TimelineComponent.tsx]: This needs to change

Comment (camilleion) [reply src/components/TimelineComponent.tsx]: Yes, it works for now however is a bit hacky, leaving myself or someone else a note for updating the data with the labels.

Comment (braunsonm) [reply src/components/TimelineComponent.tsx]: Then can you clarify that in the code? So when another developer comes across a line that says `This needs to change` we know why?

Comment (braunsonm) [reply src/components/TimelineComponent.tsx]: Oh you already did, my bad

**Pull request merged**